config/infection.json.dist

```diff
@@ -21,6 +21,48 @@
         }
     },
     "timeout": 10,
-    "minMsi": 98,
-    "minCoveredMsi": 100
+    "minMsi": 100,
+    "minCoveredMsi": 100,
+    "mutators": {
+        "@default": true,
+        "Break_": {
+            "ignore": [
+                "Psl\\Collection\\*Map::zip"
+            ]
+        },
+        "DecrementInteger": {
+            "ignore": [
+                "Psl\\DataStructure\\PriorityQueue::peek"
+            ]
+        },
+        "FunctionCallRemoval": {
+            "ignore": [
+                "Psl\\Result\\Success::getThrowable"
+            ]
+        },
+        "IncrementInteger": {
+            "ignore": [
+                "Psl\\DataStructure\\PriorityQueue::peek"
+            ]
+        },
+        "LogicalNot": {
+            "ignore": [
+                "Psl\\Hash\\Context::update"
+            ]
+        },
+        "MethodCallRemoval": {
+            "ignore": [
+                "Psl\\Iter\\Iterator::seek",
+                "Psl\\Iter\\Iterator::count"
+            ]
+        },
+        "Throw_": {
+            "ignore": [
+                "Psl\\File\\ReadHandle::__construct",
```

Comment:
I've added some tests as well - but didnt want to go into chmodding files and testing for readability / writability at this moment. 

See
https://github.com/azjezz/psl/blob/4affb62545080af0457f78a2bdcfc33cc4057f47/src/Psl/File/ReadHandle.php#L25-L35